flink-cdc-connect/flink-cdc-source-connectors/flink-cdc-base/src/main/java/org/apache/flink/cdc/connectors/base/source/metrics/SourceReaderMetrics.java

```diff
@@ -39,14 +80,23 @@ public class SourceReaderMetrics {
     /** The total number of record that failed to consume, process or emit. */
     private final Counter numRecordsInErrorsCounter;
 
+    private volatile long lastReceivedEventTime = UNDEFINED;
+    private volatile long currentReadTimestampMs = UNDEFINED;
+
     public SourceReaderMetrics(SourceReaderMetricGroup metricGroup) {
         this.metricGroup = metricGroup;
         this.numRecordsInErrorsCounter = metricGroup.getNumRecordsInErrorsCounter();
-    }
 
-    public void registerMetrics() {
         metricGroup.gauge(
                 MetricNames.CURRENT_FETCH_EVENT_TIME_LAG, (Gauge<Long>) this::getFetchDelay);
+        metricGroup.gauge(CURRENT_READ_TIMESTAMP_MS, () -> currentReadTimestampMs);
```

Comment:
`currentReadTimestampMs` is never updated.

Reply:
Yes, I have already deleted it